planner: update the fix-control doc (#19520)
pingcap/docs: optimizer-fix-controls.md

- This variable sets the threshold for the optimizer's heuristic strategy to select access paths. If the estimated rows for an access path (such as `Index_A`) is much smaller than that of other access paths (default `1000` times), the optimizer skips the cost comparison and directly selects `Index_A`.
- `0` means to disable this heuristic strategy.

### [`45798`](https://github.com/pingcap/tidb/issues/45798) <span class="version-mark">New in v7.5.0</span>

- Default value: `ON`
- Possible values: `ON`, `OFF`

> - This variable controls whether to allow the Plan Cache to cache plans accessing generated columns.

Reviewer: ```suggestion
- This variable controls whether Plan Cache is allowed to cache execution plans that access [generated columns](/generated-columns.md).
```

> ### [`46177`](https://github.com/pingcap/tidb/issues/46177) <span class="version-mark">New in v7.5.0</span>

Reviewer: ```suggestion
### [`46177`](https://github.com/pingcap/tidb/issues/46177) <span class="version-mark">New in v6.5.6, v7.1.3 and v7.5.0</span>
```

- Default value: `ON`
- Possible values: `ON`, `OFF`

> - This variable controls whether to explore enforced plans during optimization if it has already found an unenforced plan.

Reviewer: ```suggestion
- This variable controls whether the optimizer explores enforced plans during query optimization after finding an unenforced plan.
```
